migrations/versions/1ec25c296a6d_add_service_terrtory_ferc714_state_.py

```diff
@@ -0,0 +1,99 @@
+"""Add service terrtory, FERC714, state demand assets
```

Comment:
@bendnorman This will be incorrect since we rebased dev, but I'll fix this all with one migration when this gets merged into the Census PR.